packages/web/app/components/Field/LocationField.jsx

```diff
@@ -77,9 +76,8 @@ export const LocationInput = React.memo(
     }, [initialValues, name]);
 
     useEffect(() => {
-      if (value) {
-        setLocationId(value);
-      }
+      setLocationId(value ?? '');
+      if (!value) setGroupId('');
```

Comment:
I think there is a conflict with a change Rohan is doing [here](https://github.com/beyondessential/tamanu/pull/8604/files). not sure if you are aware of that?